libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKOAuth2.h

```diff
@@ -78,6 +78,13 @@ typedef NS_ENUM(NSInteger, SFLogoutReason) {
     SFLogoutReasonRefreshTokenRotated // "Refresh token rotated"
 };
 
+typedef NS_ENUM(NSInteger, SFLoginReason) {
+    SFLoginReasonUnknown,
+    SFLoginReasonRestAPI,
```

Comment:
When would it be SFLoginReasonRestAPI?

Reply:
I assumed that meant we try to refresh the access token but the auth token is expired.

Reply:
Whoops, I missed a file, for this one https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/14df0e9b53465fb6b9780934227637df398bcd0f/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m#L247

Reply:
better naming ideas would be great :P